### PR TITLE
add support for reviewed-preprint

### PR DESCRIPTION
**Pull request by smoqadam**

### CODE REVIEW

File: src/snippets/article-vor.v1.yaml [hunk 1]
```diff
@@ -14,3 +14,13 @@ allOf:
         figuresPdf:
             type: string
             format: uri
+        reviewedDate:
```
Comment (nlisgo) [inline src/snippets/article-vor.v1.yaml:17]: @lsh-0 a couple of optional parameters are being added to vor but we are not expecting these values to be available on the article store for now. We will be introducing these values will be handled in a similar way to how we are introducing executable version date to vor. It is a task for the future (as volume increases) to determine how to make these available in an automated way.

File: src/api.raml [hunk 2]
```diff
@@ -1842,6 +1901,20 @@ traits:
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
                                 value: !include samples/search/v1/empty.json
+                    application/vnd.elife.search+json;version=2:
+                        (experimental):
+                        schema: !include ../dist/model/search.v2.json
+                        examples:
```
Comment (lsh-0) [inline src/api.raml:1907]: `paginated` and `empty` examples are pretty useless.

What would be *much* more helpful is a `description` of how `v2` differs from `v1`, especially if clients consuming search results need to migrate.

File: src/model/reviewed-preprint.v1.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Reviewed preprint post
+type: object
+allOf:
+  - $ref: ../snippets/reviewed-preprint.v1.yaml
+  - properties:
+      indexContent:
+        description: Content to be indexed
```
Comment (lsh-0) [inline src/model/reviewed-preprint.v1.yaml:8]: this is an odd description for a 'reviewed preprint' contnet type

File: src/samples/article-vor/v6/15691.json [hunk 4]
```diff
@@ -4144,5 +4144,7 @@
     },
     "stage": "published",
     "statusDate": "2016-08-02T00:00:00Z",
-    "versionDate": "2020-05-14T15:04:33Z"
+    "versionDate": "2020-05-14T15:04:33Z",
+    "reviewedDate": "2020-05-14T15:04:33Z",
+    "curationLabels": ["Ground-breaking", "Convincing"]
```
Comment (lsh-0) [inline src/samples/article-vor/v6/15691.json:4149]: will case matter? all lowercase seems sensible if these are single value labels/tags.

File: src/samples/search/v2/empty.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (lsh-0) [inline src/samples/search/v2/empty.json:1]: I take it back, this `empty` example result seems helpful

File: src/snippets/article-vor.v1.yaml (hunk 1 above)
Comment (lsh-0) [reply src/snippets/article-vor.v1.yaml:17]: Is there a coordination ticket for this?

https://github.com/elifesciences/issues/blob/master/.github/ISSUE_TEMPLATE/article--feature.md

**Comment by lsh-0** [comment]
A few minor points of feedback I'd wished you'd addressed @nlisgo before you merged. We all rely on `api-raml` heavily so making changes and merging without soliciting review is not cool.

**Comment by lsh-0** [comment]
I'm grumpy today, I do appreciate your work putting all this together @nlisgo , `api-raml` is complex and fiddly.

**Pull request merged**